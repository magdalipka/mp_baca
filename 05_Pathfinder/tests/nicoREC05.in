10 10
1 0 1 0 0 1 0 1 0 0
0 0 1 0 1 0 0 0 1 0
0 1 0 0 1 0 1 0 0 0
0 0 1 0 0 0 1 0 1 1
1 0 0 0 1 0 1 0 1 0
0 0 1 1 0 0 1 0 1 0
0 1 0 0 0 1 1 0 0 0
1 1 0 1 1 1 0 0 1 0
0 0 0 1 0 0 0 1 1 0
0 1 1 1 1 1 1 0 0 0
1
r 4 9 4 1

