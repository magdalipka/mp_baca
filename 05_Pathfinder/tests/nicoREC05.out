r W S S S E E N N E E S S S S S S W S W W
